sdk/src/sysvars/rent.rs

```diff
@@ -194,25 +252,32 @@ impl Rent {
     /// # Returns
     ///
     /// The minimum balance in lamports for rent exemption.
-    #[inline]
-    #[allow(deprecated)]
-    pub fn minimum_balance(&self, data_len: usize) -> u64 {
-        let bytes = data_len as u64;
-        let exemption_threshold_as_u64 = u64::from_le_bytes(self.exemption_threshold.to_le_bytes());
+    ///
+    /// # Errors
+    ///
+    /// Returns `ProgramError::InvalidArgument` if `data_len` exceeds the maximum permitted
+    /// data length.
+    #[allow(clippy::collapsible_if)]
```

Comment:
Clippy complains about the `if` statement chain, but it was the most CU-wise efficient way to write it.